deploy/cephcsi/image/Dockerfile

```diff
@@ -50,6 +59,13 @@ RUN make cephcsi
 #-- Final container
 FROM ${BASE_IMAGE}
 
+# FIXME: Ceph base containers should use CentOS Stream 8
```

Comment:
@nixpanic this change is not required here as it's a final container and we don't install anything on it?

Reply:
We may need it, depending on the libraries that are linked against the executable. Better use the same versions of libraries for building and during runtime.

This will use the cached image created above, the commands are in the same sequence, so there is no need to download/migrate again.